components/stripe/actions/confirm-payment-intent/confirm-payment-intent.js

```diff
@@ -0,0 +1,63 @@
+const pick = require("lodash.pick");
+const stripe = require("../../stripe.app.js");
+
+module.exports = {
+  key: "stripe-confirm-payment-intent",
+  name: "Confirm a Payment Intent",
+  version: "0.0.1",
+  description: "Confirm that your customer intends to pay with current or provided payment " +
+    "method. Upon confirmation, Stripe will attempt to initiate a payment.",
+  props: {
+    stripe,
+    id: {
+      propDefinition: [
+        stripe,
+        "payment_intent",
+      ],
+      required: true,
+    },
+    customer: {
```

Comment:
Is this used?

Reply:
@dylburger good catch, and yes, but not in an obvious place, so I added an explanatory comment.